bluebottle/analytics/signals.py

```diff
@@ -0,0 +1,91 @@
+import re
+
+from django.utils import timezone
+from django.conf import settings
+from django.utils.translation import ugettext as _
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.db import connection
+
+from .utils import queue_analytics_record
+
+
+@receiver(post_save, weak=False, dispatch_uid='model_analytics')
+def post_save_analytics(sender, instance, **kwargs):
+    if not getattr(settings, 'ANALYTICS_ENABLED', False):
+        return
+
+    # Return early if instance is a migration.
+    if instance.__class__.__name__ == 'Migration':
```

Comment:
What part of the metrics could be out of sync? This is only ignoring signals from the Migration class as these classes never generate any metrics data. They would exit later anyway when checking further down for instance.Analytics but I thought it is more effiicient to just exit early.